src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -3,30 +3,122 @@
 					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
```

Comment:
@Arieldelossantos the UWP sample app seems to be crashing on startup with these changes

Reply:
problem solved, the issue was caused for using ThemeResources on a converter... for some reason it crash... 
for now what i did was extracted the 2 resources the converter needs and kept using staticResource for only those 2